env-setup/hacks/setup-gl

```diff
@@ -3,4 +3,4 @@ set -x
 
 export __EGL_VENDOR_LIBRARY_DIRS=$SNAP/etc/glvnd/egl_vendor.d:$SNAP/usr/share/glvnd/egl_vendor.d
 export LIBGL_DRIVERS_PATH=$SNAP/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}/dri
-export LD_LIBRARY_PATH=${SNAP}/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}:${SNAP}/lib/${SNAPCRAFT_ARCH_TRIPLET}
+export LD_LIBRARY_PATH=${SNAP}/usr/lib/${SNAPCRAFT_ARCH_TRIPLET}:${SNAP}/lib/${SNAPCRAFT_ARCH_TRIPLET}:$LD_LIBRARY_PATH
```

Comment:
Need to check that `LD_LIBRARY_PATH` is set before appending it.

Reply:
Not really, a trailing colon doesn't break anything… But can do if you say so.

Reply:
Somewhere I saw something like:

```
export LD_LIBRARY_PATH=/foo/bar:${LD_LIBRARY_PATH}
```
is a security risk, you should be using:
```
export LD_LIBRARY_PATH=/foo/bar${LD_LIBRARY_PATH:+:$LD_LIBRARY_PATH}
```

Reply:
But I never tried to understand the logic

Reply:
Fixed.